lib/ParseSIL/ParseSIL.cpp

```diff
@@ -2767,6 +2838,83 @@ bool SILParser::parseSILInstruction(SILBuilder &B) {
     ResultVal = B.createBuiltin(InstLoc, Id, ResultTy, subs, Args);
     break;
   }
+  // SWIFT_ENABLE_TENSORFLOW
+  case SILInstructionKind::GraphOperationInst: {
+    // Parse graph operation name.
+    if (P.Tok.isNot(tok::string_literal)) {
+      P.diagnose(P.Tok, diag::expected_tok_in_sil_instr, "graph_op name");
+      return true;
+    }
+    StringRef rawString = P.Tok.getText().drop_front().drop_back();
+    Identifier name = P.Context.getIdentifier(rawString);
+    P.consumeToken(tok::string_literal);
```

Comment:
Can you use P.parseIdentifier for this?

Reply:
I don't believe so, because `parseIdentifier` doesn't handle quotations that surround a `string_literal` token.

I copied this from the [code for parsing `StringLiteralInst`](https://github.com/apple/swift/blob/9a6adcec9d09481091de9a10d8d45b4445a26995/lib/ParseSIL/ParseSIL.cpp#L2467).
It also uses `P.Tok.getText().drop_front().drop_back()`.

Reply:
Ok!